samples/graph-change-notification-team-channel-nodejs/nodejs/Assests/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-graph-change-notification-team-channel",
+    "source": "officeDev",
+    "title": "Graph Change Notification Team/Channel",
+    "shortDescription": "Graph change notification for team and channel",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/graph-change-notification-team-channel/nodejs",
+    "longDescription": [
+      "This sample demonstrates the use of Team/Channel subscription that will post notifications when user Create/Edit/Delete the team/channel."
+    ],
+    "creationDateTime": "2022-09-19",
+    "updateDateTime": "2022-09-19",
+    "products": [
+      "Teams"
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "csharp"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "netframework"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "tab"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/v-mfurquan/graph-change-notification-team-channel-nodejs/samples/graph-change-notification-team-channel-nodejs/nodejs/Images/ChangeNotifications.gif",
```

Comment:
This url should not contain the path of your branch "v-mfurquan"
It should be main branch.